### ISSUE TITLE
confirm_permissions module not found

### ISSUE THREAD

**Issue by KishorBal** [RhinoSecurityLabs/pacu#316]
I am getting "Module not found error" when trying to run the confirm_permissions module...

**Comment by KishorBal** [comment]
@cmd-ctrl-freq  I saw it is being used in the official walkthrough of pacu 
![image](https://user-images.githubusercontent.com/68984199/163096085-6694404b-614f-47a7-9ac7-7e59cee4b9b7.png)

**Comment by cmd-ctrl-freq** [comment]
Interesting! Thank you for that screenshot. It looks like this module does not currently exist in pacu, but I can do some digging to see at what point it was removed, and maybe even find out WHY it was removed (if I'm lucky).

**Comment by cmd-ctrl-freq** [comment]
@KishorBal Okay I have solved the mystery. Looks like `confirm_permissions` was the original version of the present `iam__enum_permissions`. To find this out I searched all of git for the module name using the technique described [here](https://stackoverflow.com/questions/4468361/search-all-of-git-history-for-a-string), and then read over the description to see what the behavior was. Then read over the modern version of the module to confirm that these two modules are doing the same thing. 

I'll keep this thread open for a few days in case you have questions/comments, then close it.
